README.md

```diff
@@ -90,4 +103,68 @@ Then, click on the map and a popup with the closest weather data point informati
 
 ### Explore wind turbines informations and timeseries
  
-TDB
\ No newline at end of file
+TDB
+
+### Explore feedin time series data
+
+The feedin time series data needs to be uploaded to the OEDB. Once this is finished it will be possible to present 
+the data as graph within the WAM_APP_FRED. 
+
+### Get data as .CSV file
+
+The displayed data can be downloaded as csf file. Therefore click on the button located in the control-panel
+on the left. For example, the CSV contains all the data for a single weather point selected on the map. 
+
+## Source database
+
+### Introducing the Open-Energy-Database
+
+The database which is used as main data-source for all of the topics is the open-energy-database (OEDB). 
+The OEDB is a postgreSQL database that was developed within the open_eGo project. 
+
+### Introducing the Open-Energy-Platform
+
+The OEDB is managed within the openenergy-platform (OEP). The OEP is the presentation layer for the OEDB. The OEP 
+provides the Open-Energy Dialect (OED) which is a RestFULL API. Using the OED one can query the OEDB. The OEP provides 
+the open-energy-dialect (OED) which is the main RestFull-API to query the OEDB. The data should be provided as JSON or 
```

Comment:
should or must?

Reply:
i think "must be provided as a non-proprietary format like json" is better

Reply:
```suggestion
the open-energy-dialect (OED) which is the main RESTful API to query the OEDB. The data should be provided as JSON or 
```